source/write/update.txt

```diff
@@ -0,0 +1,159 @@
+.. _c-write-update:
+
+================
+Update Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: modify, change, bulk, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to update
+documents in a MongoDB collection. You can call the ``mongoc_collection_update_one()``
+function to update a single document or the ``mongoc_collection_update_many()``
+function to update multiple documents.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB by using the following functions:
+
+- ``mongoc_collection_update_one()``, which updates *the first document* that
+  matches the search criteria
+- ``mongoc_collection_update_many()``, which updates *all documents* that match
+  the search criteria
+
+Each update function accepts the following parameters:
+
+- Collection: Specifies the collection to update
+
+- Query filter document: Specifies which collection documents to update. For
+  more information about query filters, see the 
+  :manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+  the {+mdb-server+} manual.
+
+- Update document: Specifies the **update operator**, or the kind of update to
+  perform, and the fields and values to change. For a list of update operators
+  and their usage, see the :manual:`Field Update Operators guide
+  </reference/operator/update-field/>` in the {+mdb-server+} manual.
+
+- Results location: Specifies a pointer to overwritable storage that will contain
+  operation results, or ``NULL``
+
+- Error location: Specifies a location for an error value, or ``NULL``
```

Comment:
S: since at least one description has multiple sentences, add periods to the end of all items